gateway/rdb/src/main/kotlin/org/doorip/gateway/rdb/user/UserJpaEntity.kt

```
@@ -0,0 +1,59 @@
+import org.doorip.domain.entity.UserId
+
+@Table(name = "users")
+@Entity
+internal class UserJpaEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "user_id")
+    var id: Long = 0
+
+    @Column(name = "name", columnDefinition = "varchar(255)", nullable = false)
+    lateinit var name: String
```

detekt (1.23.7)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)